packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -273,6 +275,7 @@ export const PageTreeNode = withNodeTypeRegistry(connect(
             intermediateContextPaths: selectors.UI.PageTree.getIntermediate(state),
             loadingNodeContextPaths: selectors.UI.PageTree.getLoading(state),
             errorNodeContextPaths: selectors.UI.PageTree.getErrors(state),
+            isNodeDirty: selectors.CR.Workspaces.makeIsDocumentNodeDirtySelector($get('contextPath', node))(state),
```

Comment:
oups, that was too much ;-) the selector *factory* must be in the outer function (at the old location); but it is *not allowed to take any arguments*.

The argument "contextPath" must be passed into the selector as 2nd parameter after "state" :-)

This way you did it here means we do not memoize at all